pkg/skaffold/runner/v1/new.go

```diff
@@ -139,7 +125,28 @@ func NewForConfig(runCtx *runcontext.RunContext) (*SkaffoldRunner, error) {
 		runCtx:             runCtx,
 		intents:            intents,
 		isLocalImage:       isLocalImage,
-	}, nil
+	}
+
+	logger := r.createLogger()
+
+	var deployer deploy.Deployer
+	deployer, err = getDeployer(runCtx, labeller.Labels(), logger)
+	if err != nil {
+		return nil, fmt.Errorf("creating deployer: %w", err)
+	}
+
+	builder, tester, deployer = runner.WithTimings(builder, tester, deployer, runCtx.CacheArtifacts())
+	if runCtx.Notification() {
+		deployer = runner.WithNotification(deployer)
+	}
+	rbuilder := runner.NewBuilder(builder, tagger, artifactCache, podSelectors, runCtx)
+
+	r.Builder = *rbuilder
```

Comment:
can we move the `SkaffoldRunner` initialization here and make these inline like before?

Reply:
whoops sorry, this was from a merge conflict I just resolved that went awry. updated